(BadMapError) expected a map, got: nil

In my `next-ls` log when starting Zed I get this:

```
15:15:57.136 [error] ** (BadMapError) expected a map, got: nil
    (stdlib 5.2.3) :maps.merge(%{}, nil)
    (next_ls 0.22.8) lib/next_ls.ex:1005: anonymous fn/2 in NextLS.handle_notification/2
    (elixir 1.16.3) lib/map.ex:916: Map.update!/3
    (next_ls 0.22.8) lib/next_ls.ex:1005: NextLS.handle_notification/2
    (gen_lsp 0.10.0) lib/gen_lsp.ex:478: anonymous fn/2 in GenLSP.loop/3
    (telemetry 1.2.1) /private/tmp/next-ls-20240531-10295-xqrzy1/next-ls-0.22.8/deps/telemetry/src/telemetry.erl:321: :telemetry.span/3
    (gen_lsp 0.10.0) lib/gen_lsp.ex:474: anonymous fn/7 in GenLSP.loop/3
    (gen_lsp 0.10.0) lib/gen_lsp.ex:553: GenLSP.attempt/4
```

Installed with Brew

Same here. This is how the error looks in the overall logs:

```
2024-08-13T10:11:49.212501Z [ERROR] failed to format via language server

Caused by:
    ** (CaseClauseError) no case clause matching: :timeout
        (gen_lsp 0.10.0) lib/gen_lsp.ex:353: anonymous fn/7 in GenLSP.loop/3
        (gen_lsp 0.10.0) lib/gen_lsp.ex:553: GenLSP.attempt/4
        (stdlib 6.0) proc_lib.erl:329: :proc_lib.init_p_do_apply/3
```

--- a/lib/next_ls.ex
+++ b/lib/next_ls.ex
@@ -973,6 +973,8 @@ defmodule NextLS do
             {ref, {token, "Compiled #{name}!"}}
           end
         end)
+      else
+        Map.new()
       end
 
     lsp =

